cluster-up/cluster/kind-k8s-sriov-1.17.0/deploy_sriov_operator.sh

```diff
@@ -203,14 +206,19 @@ function deploy_sriov_operator {
 
 function apply_sriov_node_policy {
   local -r policy_file=$1
-  local -r node_pf=$2
+  local -r pfs_names_json_arrays=$2
   local -r num_vfs=$3
-
-  # Substitute $NODE_PF and $NODE_PF_NUM_VFS and create SriovNetworkNodePolicy CR
-  local -r policy=$(NODE_PF=$node_pf NODE_PF_NUM_VFS=$num_vfs envsubst < $policy_file)
-  echo "Applying SriovNetworkNodeConfigPolicy:"
-  echo "$policy"
-  _kubectl create -f - <<< "$policy"
+  
+  # Format to singal array ["ens1","ens2"],["ens3"] => ["ens1","ens2","ens3"]
```

Comment:
For the case where there are more then one workers with PF attached
This PR change also supporting multiple nodes

Reply:
Sorry, I do not get it. Each array is representing a worker? And in each array we have the PF/s that have been assign to them?

Reply:
Aha.. it comes from `_kubectl get nodes`. I think it will be easier to read by handling one node at a time in the flow.
But if it cannot work, just use a for-loop, it would be easier to name things and read the "why".

Reply:
Yes we fetch the node_pfs annotation from all nodes with label `sriov: true`.
if we would have a cluster with multiple nodes where each node have more then one PF, the annotations would looke like:
  node1: `["pf1","pf2"]`
  node1: `["pf3","pf4"]`
Then in order to include all those in the sriov policy it should be formated to single json array:
  `["pf1","pf2","pf3","pf4"]`

Maybe it will be easier to understand if the formatting was next to the part where we read the annotations,and better naming